code/go/0chain.net/blobber/config.go

```diff
@@ -134,7 +134,10 @@ func setupConfig(configDir string, deploymentMode int) {
 	config.Configuration.Geolocation.Latitude = viper.GetFloat64("geolocation.latitude")
 	config.Configuration.Geolocation.Longitude = viper.GetFloat64("geolocation.longitude")
 
-	config.Configuration.ChallengeCompletionTime = viper.GetDuration("challenge_completion_time")
+	err := setCCTFromChain()
```

Comment:
Can we move this outside of setupConfig this has nothing to do with setting up the config file. Maybe have the main function call setupSettings that calls setupConfig and setCCTFromChain?